src/main/entity/User.java

```diff
@@ -0,0 +1,31 @@
+package main.entity;
+
+public class User {
```

Comment:
Why do we need User and AbstractUser at the same time?

Reply:
By separating these responsibilities, I aim to follow the Single Responsibility Principle—keeping data management (`User`) separate from behavior logic (`AbstractUser`).

Reply:
So here we needed it's better to have also separate classes depending on role, AbstractUser, Admin and Client(admin and client has their own extra functionality)